src/components/text-editor/__internal__/toolbar/toolbar-button/toolbar-button.style.js

```diff
@@ -3,7 +3,7 @@ import { isDLS } from "../../../../../utils/helpers/style-helper";
 import baseTheme from "../../../../../style/themes/base";
 import StyledIcon from "../../../../icon/icon.style";
 
-const StyledToolbarButton = styled.button`
+const StyledToolbarButton = styled.button.attrs({ type: "button" })`
```

Comment:
This already has type button in the component file, I think it should be removed from there and implemented like this so we are consistent

https://github.com/Sage/carbon/blob/adbe79f619dab19eb147f00868246066c6abcc47/src/components/text-editor/__internal__/toolbar/toolbar-button/toolbar-button.component.js#L31